magento/magento2
# Make email validation only validate @ sign presence, fixes #4547

### Description
New TLDs aren't valid because of Zend Framework Email validator, which is too strict. As some pointed out, this might not be necessary. See discussion: https://github.com/magento/magento2/pull/8420/files#r99476657

### Fixed Issues (if relevant)
1. magento/magento2#4547: Support for new `.tech` TLD and others required

### Manual testing scenarios
1. During customer registration, try to register with email test@test.swiss
2. Observe an error being thrown http://prntscr.com/er5qdb
`"Email" is not a valid hostname. 'test.swiss' looks like a DNS hostname but we cannot match the TLD against known list. 'test.swiss' looks like a local network name, which is not an acceptable format.
`

### Contribution checklist
